### PR TITLE
Improve Dynamic Bootstrapping documentation

### PR DESCRIPTION
**Pull request by jordanschalm**
Adds a page specifically for bootstrapping which covers the different bootstrapping paths, precedence, configuration options, and some example workflows.

### CODE REVIEW

File: docs/networks/node-ops/node-operation/protocol-state-bootstrap.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+#### Example 2
+Use Dynamic Startup to bootstrap your node immediately, using the most recent Root Snapshot:
+1. Ensure your database is not already bootstrapped, and no Root Snapshot file is present in the `$BOOTSTRAPDIR` folder.
+2. Add necessary flags to node startup command.
+For example, using the Flow Foundation Access Node:
+```sh
+  ... \
+  --dynamic-startup-access-address=secure.mainnet.nodes.onflow.org:9001 \
+  --dynamic-startup-access-publickey=28a0d9edd0de3f15866dfe4aea1560c4504fe313fc6ca3f63a63e4f98d0e295144692a58ebe7f7894349198613f65b2d960abf99ec2625e247b1c78ba5bf2eae \
+  --dynamic-startup-epoch-phase=1
```
Comment (jordanschalm) [inline docs/networks/node-ops/node-operation/protocol-state-bootstrap.md:146]:
```suggestion
  --dynamic-startup-epoch=1
```

File: docs/networks/node-ops/node-operation/protocol-state-bootstrap.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+util read-protocol-state snapshot -d $DATADIR --final > latest-finalized-snapshot.json
+```
+
+Retrieve the snapshot for a specific finalized block height:
+```sh
+util read-protocol-state snapshot -d $DATADIR --height 12345 > specific-height-snapshot.json
+```
+
+4. On the node you want to bootstrap, move the Root Snapshot file to `$BOOTDIR/public-root-information/root-protocol-state-snapshot.json`, where `$BOOTDIR` is the value passed to the `--bootstrapdir` flag.
+5. Start your node.
```
Comment (jordanschalm) [inline docs/networks/node-ops/node-operation/protocol-state-bootstrap.md:71]:
```suggestion
```

File: docs/networks/node-ops/node-operation/protocol-state-bootstrap.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  --dynamic-startup-access-address=secure.mainnet.nodes.onflow.org:9001 \
+  --dynamic-startup-access-publickey=28a0d9edd0de3f15866dfe4aea1560c4504fe313fc6ca3f63a63e4f98d0e295144692a58ebe7f7894349198613f65b2d960abf99ec2625e247b1c78ba5bf2eae
+```
+
+### Specifying an Epoch Phase
+
+Two flags are used to specify when to bootstrap:
+- `--dynamic-startup-epoch-phase` - the epoch phase to start up in (default `EpochPhaseSetup`)
+- `--dynamic-startup-epoch` - the epoch counter to start up in (default `current`)
+
```
Comment (vishalchangrani) [inline docs/networks/node-ops/node-operation/protocol-state-bootstrap.md:110]: > You can check the current epoch phase of the network by running [this](https://github.com/onflow/flow-core-contracts/blob/master/transactions/epoch/scripts/get_epoch_phase.cdc) script. Alternatively, you can also check the current epoch phase [here](https://app.metrika.co/flow/dashboard/network-overview) under Epoch Phase.

Comment (vishalchangrani) [inline docs/networks/node-ops/node-operation/protocol-state-bootstrap.md:110]:
```suggestion

> You can check the current epoch phase of the network by running [this](https://github.com/onflow/flow-core-contracts/blob/master/transactions/epoch/scripts/get_epoch_phase.cdc) script. Alternatively, you can also check the current epoch phase [here](https://app.metrika.co/flow/dashboard/network-overview) under Epoch Phase.

```

**Review by vishalchangrani — APPROVED** [review]

**Pull request merged**